app/views/drawings/_drawing.html.haml

```diff
@@ -55,4 +55,14 @@
             .drawing-row
               %strong Context / Story:
               = drawing.story
-            
+            %br
+            .drawing-row
+              - case drawing.image_consent
+              - when nil
+                %span.fa.fa-question
```

Comment:
It might be good to flag that the drawing is pending image consent rather than remove it, so perhaps instead of the icon, we can post-fix the sentence with "- NOT SPECIFIED" or something better?